metrics/lib/common.bash

```diff
@@ -70,14 +70,16 @@ function check_cmds()
 function init_env()
 {
 	cmd=("docker")
-	contr_running=$(docker ps -qa)
 
+	# check dependencies
 	check_cmds "${cmd[@]}"
 
-	# Verify containers running
-	if [ ! -z "$contr_running" ];then
-		docker rm -f $(docker ps -qa)
-	fi
+	# This clean up is more aggressive, this is in order to
+	# decrease the factors that could affect the metrics results.
+	kill_processes_before_start
+
+	# Remove all stopped containers
+	clean_env
 
 	# Restart services
 	systemctl restart docker
```

Comment:
Hi @MarioCarrilloA Whilst I was running up the metrics code under Jenkins, I realised the network common code does a system level activity here (`systemctl`) without necessarily having raised privs (`sudo`). This was failing for me under Jenkins as the tests are run as the Jenkins user. I had to `sudo` the whole `run_metrics_ci.sh` script to work around it, which was not ideal :-(
I think we should discuss and define what the 'rules for `sudo`' are for the metrics code base. My premise is:
- all commands in scripts that require raised privs should be run with `sudo`
- no script should require to be run, at the top level, as `sudo`

What do you think?
/cc @bwarden @chavafg

Reply:
I agree -- sudo should be applied to the specific commands where needed, and nowhere else. Definitely not at script level.

Reply:
yeah, I meant to also say:
- run all scripts at the lowest priv they need to operate, and have them elevate when and only when necessary :-)

The one place we sort of do away with this for convenience is that we presume the id running the tests is a member of the docker group so we don't have to sudo every docker command - seeing as that is our 'bread and butter', I see that as an acceptable exception :-)

Reply:
We should also report and bail early if we don't have permission for docker.

Reply:
perfect thanks :)